python/cudf/cudf/core/frame.py

```diff
@@ -49,6 +49,91 @@ def _mimic_inplace(self, result, inplace=False):
         else:
             return result
 
+    @property
+    def size(self):
```

Comment:
Can we add a similar method to `Column` as well? (in preparation for eventually exposing them as `cudf.array`

Reply:
`Column` already has a `size` property